app/eventyay/orga/views/schedule.py

```diff
@@ -512,11 +512,23 @@ def delete_handler(self, request, *args, **kwargs):
             obj = self.get_object()
             obj.deleted = True
             obj.save(update_fields=['deleted'])
-            messages.success(request, phrases.base.success)
+            messages.success(request, _('The selected room has been deleted.'))
             return redirect(self.get_success_url())
         except ProtectedError:
             messages.error(
                 request,
                 _('There is or was a session scheduled in this room. It cannot be deleted.'),
             )
             return self.delete_view(request, *args, **kwargs)
+        except Exception as e:
+            # Log the unexpected error for debugging
+            logger.exception(
+                'Unexpected error during room deletion for room %s: %s',
+                kwargs.get('pk', 'unknown'),
+                str(e)
```

Comment:
If you use `logger.exception`, don't pass the error to the message, because this `logger.exception` always capture the error and show its traceback.